src/api.v2/helpers/pipeline/pipelineConstruct/index.js

```diff
@@ -163,6 +193,8 @@ const createQCPipeline = async (experimentId, processingConfigUpdates, authJWT,
     {
       state_machine_arn: stateMachineArn,
       execution_arn: executionArn,
+      last_pipeline_params: { cellMetadataId: currentCellMetadataId },
+      retry_params: { clusteringShouldRun },
```

Comment:
It is stored here because, if the `configureEmbedding` step fails, we need to know whether the last time it was run it was running clustering or not.

An alternative to this could be having `clusteringShouldRun` in the `processingConfig` instead. I am not completely sure about this option because the differences between this prop and the rest of the processing configs still holds, but if anyone thinks that would make more sense let me know. 

Important to note, it would be stored inside the processing config as a sort of "record" of the config of the previous attempted run, but not as something that can be changed by the user in any way. It is a property calculated based on other properties. 

I didn't store it in `last_pipeline_params` because that stores the values that, when changed, cause a pipeline to rerun. This property is different in that it is changed as a consequence of a pipeline running, it doesn't cause the pipeline reruns.

Reply:
what is exactly `currentCellMetadatId` and what is used for?
regarding the `clusteringShouldRun`, I think it's fine although it's a bit hacky. Maybe we don't need to store that and if the embedding fails just re-run also the clustering. I don't think this fails often enough to have to keep track of it. I haven't run into any embeddings failure that I can remember of. Maybe @gerbeldo or @saracastel input can be useful here. I would probably just make users re-run both and avoid having to deal with this in the code.

Reply:
About this: 

> Important to note, it would be stored inside the processing config as a sort of "record" of the config of the previous attempted run, but not as something that can be changed by the user in any way. It is a property calculated based on other properties.

Why would `clusteringShouldRun` be stored in the `processingConfig`?

Reply:
> what is exactly currentCellMetadatId
- `currentCellMetadataId` is the id of the currently uploaded cell level metadata file, it is used to know whether to put Process project of Go to data exploration in the button in data management in a similar way as it is done for gem2s for the same button.

> Maybe we don't need to store that and if the embedding fails just re-run also the clustering
I guess so, the problem with that is that we fall back to the same problem as before of going to all the trouble of not overwriting the clustering just to end up doing it when something in cell level fails. I personally think we should be able to rerun a failed pipeline with the expected params rather than defaulting to something else, my question was more along the lines of visibilizing this way of doing it vs an alternative.